### ISSUE TITLE
Module 'PINOperation' not found ERROR in objc!

### ISSUE THREAD

**Issue by internalG** [pinterest/PINCache#289]
The @import PINOperation; change bring this error to project. The SPM support breaks objc project.

**Comment by Braindie** [comment]
+1

**Comment by iwheelbuy** [comment]
+1

**Comment by iwheelbuy** [comment]
I believe something is wrong with `pod 'PINCache', '3.0.2'`

```
def pod__PINRemoteImage
  pod 'PINRemoteImage', '3.0.1', :modular_headers => true
  pod 'PINCache', '3.0.1', :modular_headers => true
end
```

This way it works for me.

**Comment by garrettmoon** [comment]
What version of Xcode are you using? Can you upload a sample project demonstrating the issue?

**Comment by iwheelbuy** [comment]
@garrettmoon 12 GM or 12.0.1 - both don't work. 
[PIN.zip](https://github.com/pinterest/PINCache/files/5359612/PIN.zip)

**Comment by yang152412** [comment]
> The @import PINOperation; change bring this error to project. The SPM support breaks objc project.

+1

**Comment by luoshihui** [comment]
> The @import PINOperation; change bring this error to project. The SPM support breaks objc project.

+1

**Comment by internalG** [comment]
The podfile to reproduce the issue:

# Uncomment this line to define a global platform for your project
platform :ios, '8.0'

# ignore all warnings from all pods
inhibit_all_warnings!

target 'pname' do
pod 'AFNetworking/NSURLConnection', '~> 2.0'
pod 'PINCache'
end

target 'OneRiliTests' do

end

# fix warnings in Xcode 10
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      if config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'].to_f < 8.0
        config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '8.0'
      end
    end
  end
end

**Comment by garrettmoon** [comment]
> **UPDATE:**
> For anyone with the same question, I ran `pod update --verbose` and realized the 3.0.2 branch of PINCache was being fetched (while the fix is in `master`). Adding the following to the podfile fetches the right versions with imports fixed:
> 
> `pod 'PINCache', :git => 'https://github.com/pinterest/PINCache.git', :branch => 'master'`
> 
> @garrettmoon I saw that you fixed this issue and merged to `master` (🙏) but the update doesn't seem to reflect in Texture which specifies PINCache as a dependency. That’s still pulling the `.m` files with `@import PINOperation;`. Apologies if I'm missing something stupid. 😬

You're not missing anything – it takes us time to cascade releases all the way down to Texture. I'm working on making this easier so we take less time!

**Comment by internalG** [comment]
Fixed with 3.0.3.
